source/developer/developer-setup.md

```diff
@@ -90,6 +90,14 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+#### Build and install with AUR ####
```

Comment:
Since this package is not maintained by Mattermost, I think we need to apply an `Unofficial` tag here.

Please append `(Unofficial)` to the end of the heading?